tests/autologging/test_training_session.py

```diff
@@ -1,6 +1,8 @@
 import pytest
 
-from mlflow.sklearn import _SklearnTrainingSession
+from mlflow.utils.autologging_utils import _get_training_session
+
+_TrainingSession = _get_training_session()
```

Comment:
Maybe change the function name to be `_create_new_training_session_class` will be more clear ?
like:
```
_SklearnTrainingSession = _create_new_training_session_class()
```

Reply:
Good point. Putting `_class` would clarify what this function returs.